### PR TITLE
GoogleTranslate: idle_time setting

### PR DESCRIPTION
**Pull request by alexandr-san4ez**

### CODE REVIEW

File: GoogleTranslate/googletranslate.ini [hunk 1]
```diff
@@ -27,6 +27,12 @@
 # * Default: en
 #output_lang = en
 
+# Time that the plugin will wait before sending the request.
+# * Time in seconds (can be used with float type)
+# * The minimum value is 0.25
+# * Default: 0.25
+#waiting_time = 0.25
```
Comment (polyvertex) [inline GoogleTranslate/googletranslate.ini]: Same remarks than for WebSuggest:
* hard-coded maximum value of 3 seconds
* I think `idle_time` is more appropriate than `waiting_time`

File: GoogleTranslate/googletranslate.py [hunk 2]
```diff
@@ -227,6 +229,10 @@ def _warn_skip_custitem(name, section):
         else:
             self.default_lang_out = validated_lang_code
 
+        self.waiting_time = settings.get_float(
+            "waiting_time", self.CONFIG_SECTION_DEFAULTS,
+            fallback=self.DEFAULT_WAITING_TIME, min=0.25)
```
Comment (polyvertex) [inline GoogleTranslate/googletranslate.py]: (max value)

**Comment by polyvertex** [comment]
Please PR on the `dev` branch

**Comment by alexandr-san4ez** [comment]
Maybe you did not see the change. Check this PR and #4.

**Pull request merged**